ovn_k8s/common/kubernetes.py

```diff
@@ -105,7 +105,7 @@ def get_pod_annotations(server, namespace, pod):
         response = requests.get(url, headers=headers, verify=ca_certificate)
     else:
         response = requests.get(url, headers=headers)
-    if not response or response.status_code != 200:
+    if not response:
```

Comment:
Ideally I'd prefer this type of change to be in its own bug fix commit and not include in the unit tests commit.